### PR TITLE
lestarch: fixing problem with fpp-locs

### PR DESCRIPTION
**Pull request by LeStarch**
| | |
|:---|:---|
|**_Originating Project/Creator_**| |
|**_Affected Component_**|  |
|**_Affected Architectures(s)_**|  |
|**_Related Issue(s)_**|  |
|**_Has Unit Tests (y/n)_**|  |
|**_Builds Without Errors (y/n)_**|  |
|**_Unit Tests Pass (y/n)_**|  |
|**_Documentation Included (y/n)_**|  |

---
## Change Description

Add a wrapper to help with running `fpp-locate-defs` that performs the shell redirection.

### CODE REVIEW

**Comment by LeStarch** [comment]
@kevin-f-ortega you can try this branch if you'd like.  It is part 1 of the faster generate!

File: cmake/target/fpp_locs.cmake [hunk 1]
```diff
@@ -0,0 +1,109 @@
+####
+# fpp-locs.cmake:
```
Comment (kevin-f-ortega) [inline cmake/target/fpp_locs.cmake:2]: Update name here and throughout this file.

**Review by kevin-f-ortega — COMMENTED** [review]
LGTM. I tried this out on my side and it worked. Great job! Just a few minor comments.

File: Svc/SystemResources/CMakeLists.txt [hunk 2]
```diff
@@ -14,9 +14,11 @@ set(MOD_DEPS
   Os
 )
 register_fprime_module()
-if (NOT FPRIME_FPP_LOCS_BUILD)
+if (TARGET version)
     get_module_name(${CMAKE_CURRENT_LIST_DIR})
     add_dependencies(${MODULE_NAME} version)
+else()
+    file(WRITE "${CMAKE_BINARY_DIR}/version.hpp" "#ifndef __VERSION_HPP__\n#define __VERSION_HPP\nstatic const char* VERSION = \"NOT_AVAILABLE\"\n#endif\n;")
```
[the remark this answers is not in the document]
Comment (LeStarch) [reply Svc/SystemResources/CMakeLists.txt:21]: This is redone entirely in #1268, so I am going to ignore fixing this for now.  The version target always exists anyway, so it shouldn't run here.

**Review by kevin-f-ortega — APPROVED** [review]
LGTM!

**Pull request merged**